versioncontrol/peer.go

```diff
@@ -228,14 +230,14 @@ func (versions Versions) ValidateRollouts(log *zap.Logger) error {
 	fieldCount := value.NumField()
 	validationErrs := errs.Group{}
 	for i := 1; i < fieldCount; i++ {
-		binary, ok := value.Field(i).Interface().(Binary)
+		binary, ok := value.Field(i - 1).Interface().(Binary)
```

Comment:
I'm wondering why the last field (i.e `fieldCount - 2`) isn't considered.

Reply:
I think this is just an oversight, I think `i` should just start at 0.